[BUG] Fix type errors: "Expected 0 arguments, but got 1" (TS2554)

**Describe the bug**
The codebase has several places where functions are being called with arguments, but their TypeScript definitions expect zero arguments, resulting in TS2554 errors during type checking.

**To reproduce**
Steps to reproduce the behavior:
1. Run `npm run typecheck`
2. Observe the TS2554 errors for functions expecting 0 arguments but receiving 1.

**Error output**
```
Error: src/pages/api/dataset/evaluate.ts(113,5): error TS2554: Expected 0 arguments, but got 1.
Error: src/server/api/routers/limits.ts(47,64): error TS2554: Expected 0 arguments, but got 1.
Error: src/server/background/workers/evaluationsWorker.ts(390,5): error TS2554: Expected 0 arguments, but got 1.
Error: src/server/topicClustering/topicClustering.ts(55,5): error TS2554: Expected 0 arguments, but got 1.
```

**Expected behavior**
No TypeScript errors when running type checks; all function calls match their declared signatures.

**Additional context**
You'll need to either update the function calls to remove unnecessary arguments *or* update the function definitions to accept an argument as appropriate, so that all usages match the function signatures and type checking passes.

Failing CI:
- https://github.com/langwatch/langwatch/actions/runs/19642102945/job/56248028303?pr=870

--- a/langwatch/src/server/api/routers/limits.ts
+++ b/langwatch/src/server/api/routers/limits.ts
@@ -191,8 +191,16 @@ const getCurrentMonthCostForProjects = async (projectIds: string[]) => {
   );
 };
 
-export const maxMonthlyUsageLimit = async (organizationId: string) => {
-  const plan =
-    await dependencies.subscriptionHandler.getActivePlan(organizationId);
-  return plan.maxMessagesPerMonth;
+/**
+ * Get the maximum monthly usage limit for the organization.
+ * FIXME: This was recently changed to return Infinity,
+ * but still takes the organizationId as a parameter.
+ *
+ * Either we remove the organizationId parameter from all the calls to this function,
+ * or we use to get the plan and return it correctly.
+ *
+ * @returns The maximum monthly usage limit for the organization.
+ */
+export const maxMonthlyUsageLimit = async (_organizationId: string) => {
+  return Infinity;
 };

